contrib/test-miri.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Maybe `#!/usr/bin/env bash`? Who knows what `bash` is available and which version or where in GH CI?

Reply:
I personally had more problems with `env` rather than not using `env` but it's a policy here to use env so I'm going to change it. Thanks for pointing that out!